0.13/build.sbt

```diff
@@ -27,7 +31,7 @@ def commonSettings =  Seq(
           <url>http://typosafe.pl</url>
         </developer>
       </developers>)
-)
+) ++ dependOnSbt("0.13.15")
```

Comment:
I think you don't need this, sbt 1.0.0 and 0.13.16-M1 have built-in cross-build support for sbt, so you can do `^ scripted` and it will run scripted for all the sbt versions. If you want to switch to a concrete version, use `^^ sbtVersion`.